Content.Shared/Drunk/DrunkSystem.cs

```diff
@@ -19,6 +20,9 @@ public abstract class SharedDrunkSystem : EntitySystem
         if (applySlur)
             _slurredSystem.DoSlur(uid, TimeSpan.FromSeconds(boozePower), status);
 
+        if (EntityManager.HasComponent<LightweightComponent>(uid))
```

Comment:
You should use the proxy method here.

```suggestion
        if (HasComp<LightweightComponent>(uid))
```